{"uid":"8f5e08e0f4f7ac0","name":"test_platform","fullName":"Test_Cases.test_platform.TestPlatform#test_platform","historyId":"089703e47fa4f03c2912a566c93ad585","time":{"start":1690971483385,"stop":1690971483385,"duration":0},"status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip","statusTrace":"request = <SubRequest 'get_webdriver' for <Function test_homepage>>\nget_chrome_options = <selenium.webdriver.chrome.options.Options object at 0x7f536fdac640>\n\n    @pytest.fixture(scope='session')\n    def get_webdriver(request, get_chrome_options):\n        options = get_chrome_options\n>       driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)\n\nTest_Cases/conftest.py:34: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/chrome.py:39: in install\n    driver_path = self._get_driver_path(self.driver)\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/download_manager.py:28: in download_file\n    response = self._http_client.get(url)\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py:33: in get\n    self.validate_response(resp)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [404]>\n\n    @staticmethod\n    def validate_response(resp: requests.Response):\n        status_code = resp.status_code\n        if status_code == 404:\n>           raise ValueError(f\"There is no such driver by url {resp.url}\")\nE           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip\n\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py:16: ValueError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_chrome_options","time":{"start":1690971482595,"stop":1690971482595,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get_webdriver","time":{"start":1690971482596,"stop":1690971483187,"duration":591},"status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/LTA_TESTS/LTA_TESTS/Test_Cases/conftest.py\", line 34, in get_webdriver\n    driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/chrome.py\", line 39, in install\n    driver_path = self._get_driver_path(self.driver)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/manager.py\", line 30, in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/download_manager.py\", line 28, in download_file\n    response = self._http_client.get(url)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py\", line 33, in get\n    self.validate_response(resp)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py\", line 16, in validate_response\n    raise ValueError(f\"There is no such driver by url {resp.url}\")\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"parentSuite","value":"Test_Cases"},{"name":"suite","value":"test_platform"},{"name":"subSuite","value":"TestPlatform"},{"name":"host","value":"fv-az563-955"},{"name":"thread","value":"2385-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Test_Cases.test_platform"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":6,"unknown":0,"total":7},"items":[{"uid":"f7fba71f01d4862d","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/32//#testresult/f7fba71f01d4862d","status":"passed","time":{"start":1682507322438,"stop":1682507327429,"duration":4991}},{"uid":"e1aa7b1036aa8a40","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/32//#testresult/e1aa7b1036aa8a40","status":"passed","time":{"start":1682505903845,"stop":1682505908472,"duration":4627}},{"uid":"fefc9db49077ac7b","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/31//#testresult/fefc9db49077ac7b","status":"passed","time":{"start":1682504152272,"stop":1682504157376,"duration":5104}},{"uid":"9fb5079ea1cada3c","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/30//#testresult/9fb5079ea1cada3c","status":"passed","time":{"start":1682503627546,"stop":1682503633707,"duration":6161}},{"uid":"35587a2ece4ca77b","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/29//#testresult/35587a2ece4ca77b","status":"passed","time":{"start":1682502950875,"stop":1682502955370,"duration":4495}},{"uid":"1811936264309585","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/28//#testresult/1811936264309585","status":"passed","time":{"start":1682500790654,"stop":1682500794897,"duration":4243}}]},"tags":[]},"source":"8f5e08e0f4f7ac0.json","parameterValues":[]}